### ISSUE TITLE
@react-keycloak/ssr has no exported member for ServerPersistors

### ISSUE THREAD

**Issue by kriswuollett** [panz3r/react-keycloak#75]
I've been able to successfully use `@react-keycloak/nextjs` with a locally running Keycloak server, but I'd like to use server-side rendering to avoid checking for variables like `keycloakInitialized` to avoid partial updates on page load for things like the `welcomeMessage` in the sample (parsed token isn't available the same time as authenticated).

I followed the directions for using ssr, but it seems like the members aren't been exported correctly? The sample code in the instructions
[does not import from `@react-keycloak/ssr`](https://github.com/panz3r/react-keycloak/commit/59bcaea0adba45043236cfd160dc81bf9376f6e2#diff-28dc65170961fe4d00c7d144b7066f47R84) unlike the example for Razzle below it.

VSCode doesn't show any errors regarding the imports (after my changes), but fails at runtime. I'm quite new to JavaScript/TypeScript so not sure how to debug that difference, and this issue in general. 

**To Reproduce**

I've patched the `nextjs-app` example in an attempt to get it to use ssr. Check out the [`ssr-example` branch of my `react-keycloak-examples` fork](https://github.com/kriswuollett/react-keycloak-examples/tree/ssr-example) and run `npm run dev`.

**Expected behavior**

Index page renders.

**Actual behavior**

VSCode shows no errors, but the page fails to load and this appears in the terminal:

```
error - ./pages/_app.tsx
Attempted import error: 'Persistors' is not exported from '@react-keycloak/ssr'.
```

Forcing reload on the page also yields:

```
ReferenceError: window is not defined
    at Object.<anonymous> (/Users/kris/workspace/react-keycloak-examples/examples/nextjs-app/node_modules/keycloak-js/dist/keycloak.js:63:4)
    at Module._compile (internal/modules/cjs/loader.js:1200:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1220:10)
    at Module.load (internal/modules/cjs/loader.js:1049:32)
    at Function.Module._load (internal/modules/cjs/loader.js:937:14)
    at Module.require (internal/modules/cjs/loader.js:1089:19)
    at require (internal/modules/cjs/helpers.js:73:18)
    at Object.<anonymous> (/Users/kris/workspace/react-keycloak-examples/examples/nextjs-app/node_modules/@react-keycloak/ssr/dist/lib/react-keycloak-ssr.js:15:34)
    at Module._compile (internal/modules/cjs/loader.js:1200:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1220:10)
    at Module.load (internal/modules/cjs/loader.js:1049:32)
    at Function.Module._load (internal/modules/cjs/loader.js:937:14)
    at Module.require (internal/modules/cjs/loader.js:1089:19)
    at require (internal/modules/cjs/helpers.js:73:18)
    at eval (webpack-internal:///@react-keycloak/ssr:1:18)
    at Object.@react-keycloak/ssr (/Users/kris/workspace/react-keycloak-examples/examples/nextjs-app/.next/server/static/development/pages/_app.js:127:1)
```

**Comment by panz3r** [comment]
Hi @kriswuollett ,

Thanks for your feedback, it seems there's an error in the docs, for the `ssr` package the correct import for the `SSRCookies` persistor is from `ServerPersistors`, like this

```jsx
import { ServerPersistors } from '@react-keycloak/ssr';

    <SSRKeycloakProvider
      keycloakConfig={keycloakCfg}
      persistor={ServerPersistors.SSRCookies(cookies)}
    >
```

**Comment by danielaQVMuniz** [comment]
I have the same issue when importing:
```
import {
  KeycloakCookies,
  SSRKeycloakProvider,
  ServerPersistors,
} from '@react-keycloak/ssr'
```
I get the message:
```
Module '"../../node_modules/@react-keycloak/ssr"' has no exported member 'ServerPersistors'.
```

**Comment by kriswuollett** [comment]
Hi @panz3r, please check out my `package.json` and other configuration details in the `window` not defined issue mention above. I've tried both `keycloak-js` `9.0.3` and `10.0.2`.  I can't get `@react-keycloak/ssr` to work on nextjs, but `@react-keycloak/nextjs` works fine.

**Comment by mpranjic** [comment]
It seems the code is exporting `ServerPersistors` but [Typescript definition](https://github.com/panz3r/react-keycloak/blob/master/packages/ssr/index.d.ts) is not updated

**Comment by panz3r** [comment]
Hi,

Thanks for your feedbacks, TS definitions have been fixed as of version `2.0.0-alpha.2` of `@react-keycloak/ssr`
